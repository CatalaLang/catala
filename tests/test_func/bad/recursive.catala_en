## Article

```catala
declaration scope RecursiveFunc:
  context f content integer depends on integer

scope RecursiveFunc:
  definition f of x equals f of x + 1
```
