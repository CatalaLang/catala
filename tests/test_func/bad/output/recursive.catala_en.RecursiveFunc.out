[ERROR] The variable f is used in one of its definitions, but recursion is forbidden in Catala
[ERROR] 
[ERROR]  --> test_func/bad/recursive.catala_en
[ERROR]   | 
[ERROR] 8 |   definition f of x equals f of x + 1
[ERROR]   |                            ^
[ERROR]   + Article
